front/components/assistant_builder/types.ts

```diff
@@ -229,25 +107,25 @@ export type AssistantBuilderActionAndDataVisualizationConfiguration =
   | AssistantBuilderDataVisualizationConfiguration;
 
 export type TemplateActionType = Omit<
-  AssistantBuilderActionConfiguration,
+  AssistantBuilderMCPConfiguration,
   "configuration"
 > & {
   help: string;
 };
 
-export type AssistantBuilderActionType =
-  AssistantBuilderActionConfiguration["type"];
+export type AssistantBuilderMCPServerType =
+  AssistantBuilderMCPConfiguration["type"];
 
 export type AssistantBuilderDataVisualizationType =
   AssistantBuilderDataVisualizationConfiguration["type"];
 
-export type AssistantBuilderActionState =
+export type AssistantBuilderMCPOrVizState =
```

Comment:
nitpick but I think it's cleaner to say AssistantBuilderActionState, AssistantBuilderConfiguration etc, but do we want to explicitly say it's "MCP"? Is it also to explicitly say Viz too? 🤔